env/prepare.go

```diff
@@ -0,0 +1,65 @@
+package env
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// PrepareWindowsOSEnv merges map of Windows registry environment variables with in-memory environment vars
+// Injects os.Getenv an os.Setenv
+func PrepareWindowsOSEnv(regEnvMap map[string]string, getenv func(string) string, setenv func(string, string) error) error {
```

Comment:
Injects `setenv` to avoid test pollution and `getenv` to allow testing for the former.